src/main/kotlin/systems/ajax/englishstudytelegrambot/repository/LibraryRepository.kt

```diff
@@ -1,6 +1,83 @@
 package systems.ajax.englishstudytelegrambot.repository
 
-import org.springframework.data.mongodb.repository.MongoRepository
+import org.bson.types.ObjectId
+import org.springframework.dao.DuplicateKeyException
+import org.slf4j.LoggerFactory
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.query.Criteria
+import org.springframework.data.mongodb.core.query.Query
+import org.springframework.stereotype.Repository
 import systems.ajax.englishstudytelegrambot.entity.Library
+import systems.ajax.englishstudytelegrambot.entity.Word
+import systems.ajax.englishstudytelegrambot.exception.LibraryWithTheSameNameForUserAlreadyExistException
+import systems.ajax.englishstudytelegrambot.exception.LibraryIsMissingException
 
-interface LibraryRepository : MongoRepository<Library, String>
+interface LibraryRepository {
+
+    fun saveNewLibrary(nameOfNewLibrary: String, telegramUserId: String): Library
+
+    fun getAllLibraries(): List<Library>
+
+    fun deleteLibrary(nameOfLibraryForDeleting: String, telegramUserId: String): Library
+
+    fun getLibraryByPairLibraryNameAndTelegramUserId(libraryName: String, telegramUserId: String): Library
+
+    fun getLibraryIdByPairLibraryNameAndTelegramUserId(libraryName: String, telegramUserId: String): String
```

Comment:
```suggestion
    fun getLibraryByLibraryNameAndTelegramUserId(libraryName: String, telegramUserId: String): Library

    fun getLibraryIdByLibraryNameAndTelegramUserId(libraryName: String, telegramUserId: String): String
```
no need to write Pair, you have already listed arguments in method name